src/Explorer/Tabs/DocumentsTab.ts

```diff
@@ -923,10 +915,7 @@ export default class DocumentsTab extends TabsBase {
       onCommandClick: () => {
         const selectedCollection: ViewModels.Collection = container.findSelectedCollection();
         const focusElement = document.getElementById("itemImportLink");
-        const uploadItemsPane = container.isRightPanelV2Enabled()
```

Comment:
Does anyone else use this? Are you able to safely get rid of it too?

Reply:
No, I have removed